src/sentry/mail/notifications.py

```diff
@@ -59,7 +65,17 @@ def can_users_unsubscribe(notification: BaseNotification) -> bool:
 
 
 def log_message(notification: BaseNotification, user: User) -> None:
-    pass
+    if isinstance(notification, AlertRuleNotification):
```

Comment:
should there be a log for ActivityNotifications?

Reply:
We aren't currently logging for ActivityNotifications but I'll add it in.

Reply:
will it be added in this PR or later on?

Reply:
Just added `ActivityNotification`.